.github/workflows/deno.yml

```
@@ -0,0 +1,32 @@
+name: Deno
+
+on:
+  push:
```

This causes CI to run twice when you branch and PR.
```suggestion
  push:
    branches:
      - 'main'
```

---

yeah… the alternative is to not get CI when using branches without a PR… which I personally do regularly while experimenting with stuff…
Both ways are not great…